app/frontend/css/application.scss

```diff
@@ -265,22 +265,23 @@ $govuk-global-styles: true; // https://frontend.design-system.service.gov.uk/sas
     th[aria-sort="ascending"] a,
     th[aria-sort="descending"] a {
       position: relative;
+      display: inline-block;
     }
 
     th[aria-sort="none"] a {
 
       &::before {
         content: " ▼";
         position: absolute;
-        right: -9px;
```

Comment:
I assume you've tried these CSS changes on the other sortable tables (e.g. 'delete old dogs 1 + 2') so there is no negative impact?